vendors/nxp/boards/lpc54018iotmodule/aws_demos/config_files/http_demo_s3_download_multithreaded_config.h

```diff
@@ -123,7 +123,7 @@
  * be less than democonfigUSER_BUFFER_LENGTH. We don't expect S3 to send more
  * than 1024 bytes of headers.
```

Comment:
If S3 cannot send more than 1024 bytes of header, shouldn;t the buffer be larger than that to leave space for the response body?

Reply:
Yea. Talked to @sukhmanm and updated the config